rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -93,7 +93,7 @@ def required_components(cls) -> List[Type[Component]]:
         # Dimension size of embedding vectors
         EMBEDDING_DIMENSION: 20,
         # Default dense dimension to use if no dense features are present.
-        DENSE_DIMENSION: {TEXT: 512, LABEL: 20},
+        DENSE_DIMENSION: {TEXT: 256, LABEL: 20},
```

Comment:
don't we want to change it in diet as well?

Reply:
I haven't tested for DIET.

Reply:
@vova Tested this on all datasets. Results aren't consistent and mostly degrading. Let's keep this as it is.

Reply:
I mean for DIET. Let's not change there.

Reply:
yes, let's change in diet, but keep in embedding intent classifier

Reply:
Okay, sorry if there was some confusion in my last comments. Results are inconclusive and mostly degrading for DIET, so we should not make this change in DIET. Whereas for EmbeddingIntentClassifier, making this change helps consistently across all datasets.